Bio_stats_assignment.py

```diff
@@ -141,3 +142,68 @@ def sick_patients(
     return unique_patients_  # O(1)
 
     # complexity of the above function is O(2+1+1+N+N+N+1) -> O(5+3N) -> O(N)
+
+
+def patient_age(
```

Comment:
I appreciate the thorough comments. The function name could be more informative. In my mind, "patient age" is just their current age...